readme.md

```diff
@@ -94,7 +93,7 @@ Plugins live in the `server/plugins` directory.
 
 The [good](https://github.com/hapijs/good) and [good-console](https://github.com/hapijs/good-console) plugins are included and configured in `server/plugins/logging`
 
-The logging plugin is only registered in when `NODE_ENV=development`.
+The logging plugin is only registered in development builds.
```

Comment:
This file should be renamed README.md, and there are badges at the top of the file that refer to a different repository.

Reply:
I'll sort these issues separately as they're not related to this PR.